tests/common/plugins/sanity_check/__init__.py

```diff
@@ -269,6 +272,20 @@ def sanity_check(localhost, duthosts, request, fanouthosts, nbrhosts, tbinfo):
             pt_assert(False, "!!!!!!!!!!!!!!!! Post-test sanity check failed: !!!!!!!!!!!!!!!!\n{}"\
                 .format(json.dumps(post_failed_results, indent=4, default=fallback_serializer)))
 
+        logger.info("Compare with golden config db")
+        for duthost in duthosts:
+            if duthost.stat(path=GOLDEN_CONFIG).get('stat', {}).get('exists', False):
+                running_config = duthost.get_running_config_facts()
+                logger.debug("Running config of dut {}".format(running_config))
+
+                result = duthost.command("cat {}".format(GOLDEN_CONFIG))
+                golden_config = json.loads(result['stdout'])
+                logger.debug("Golden config of dut {}".format(golden_config))
```

Comment:
The config db is pretty long. The code here will generate too much log. Even at debug level, too much log can cause significant negative impact to debugging experience.

Reply:
We should always pay attention to the logs generated by scripts, especially the common code that is used by all test scripts. The debug level log should be detailed enough for later troubleshooting. And the log should also be concise enough to be readable. It the log is filled with too much repetitive or irrelevant messages, it would be painful to dig useful information. There is a tradeoff to make.

Reply:
You are right, the log here is too long, I will remove the log here.